src/main/java/HorizontalSeamCreator.java

```diff
@@ -0,0 +1,30 @@
+public class HorizontalSeamCreator {
+
+    public HorizontalSeamCreator() {
+    }
+
+    public int[] createSmallestHorizontalSeam(int[][] energyGrid) {
+        ArrayInverter arrayInverter = new ArrayInverter();
+        int[][] invertedEnergyGrid = arrayInverter.invert2DIntArray(energyGrid);
+        int height = invertedEnergyGrid.length;
+        int width = invertedEnergyGrid[0].length;
+        PointsAndPositions pointsAndPositions = new PointsAndPositions(invertedEnergyGrid, height, width);
```

Comment:
What are points and positions?

Reply:
Points is the array of the points of the shortest path from each position at the current row the program is at while creating a path. Positions is the 2d array that holds the moves for the path that can be traced back to recreate the shortest path. Does this need to be changed? Thank you!